CMakeLists.txt

```diff
@@ -1315,8 +1346,22 @@ set(GPU_D3D11
 	GPU/D3D11/VertexShaderGeneratorD3D11.h
 )
 
-# We build Vulkan even on Apple to avoid annoying build differences.
```

Comment:
*even on Apple and WiiU to avoid annoying build differences?

-[Unknown]

Reply:
I'll give it another shot, I got some errors the first time that's why I went with the easier solution.